content/docs/guide.md

```diff
@@ -68,10 +68,26 @@ If none of the above work for you, try running it in a container
 from our Fedora based image. It contains packit installed from `main` branch,
 i.e. the same you'd get by pip installing from Github.
 
-    $ podman run -ti --rm -v $PWD:/src --security-opt label=disable quay.io/packit/packit bash
+    $ podman run -ti --rm -v $PWD:/src:z quay.io/packit/packit bash
     $ packit
     Usage: packit [OPTIONS] COMMAND [ARGS]...
 
+Based on the command you need to perform you should configure your container with the needed 
+secrets and configuration files like in the following examples:
+
+#### packit copr-build
+
+    $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr:z -v $PWD:/src:z quay.io/packit/packit bash
+    $ packit copr-build
+
+#### packit build
+
+    $ podman run -ti --rm -v ~/.ssh/:/root/.ssh:z -v $PWD:/src:z quay.io/packit/packit bash
+    $ export KRB5CCNAME=/tmp/ccache
```

Comment:
I've opened packit/packit#1492 :boat: so that this can be dropped in the future.

Reply:
I saw that, really nice! But what do you mean in the future? Should we wait to release this PR until the image will be available to the users?

Reply:
It's already available, so you could remove this :)

```
$ podman run -it --rm --pull=always quay.io/packit/packit bash -c 'echo $KRB5CCNAME'
...
/tmp/ccache
```